system/lib/libc/emscripten_time.c

```diff
@@ -25,6 +25,13 @@ void _localtime_js(const time_t *restrict t, struct tm *restrict tm);
 void _gmtime_js(const time_t *restrict t, struct tm *restrict tm);
 double _emscripten_date_now();
 double emscripten_get_now_res();
+void emscripten_get_timezone_js(char* buffer, int length);
+
+char* emscripten_get_timezone() {
+  static char buffer[128];
```

Comment:
Maybe we should go with `thread_local` here rather than static?  (for single threaded builds it will be the same anyway).